runtime/binding-grpc/src/main/java/io/aklivity/zilla/runtime/binding/grpc/internal/stream/GrpcServerFactory.java

```diff
@@ -171,7 +172,7 @@ public void doNetAbort(
                 long authorization,
                 String16FW status)
             {
-                server.doGrpcNetStatus(traceId, authorization, status);
+                server.doGrpcNetStatus(traceId, authorization, server.replyPadding, status);
```

Comment:
Is this correct? (padding for reserved as parameter)

Reply:
No, it is not correct it should be 0 as we are handeling abort which was not claimed anywhere. Thanks for the catch.